js/bibox.js

```diff
@@ -428,24 +390,24 @@ module.exports = class bibox extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
```

Comment:
@samgermain the numbers you're adding here don't seem to match the current precision mode (TICK_SIZE)

Reply:
can you be more descriptive?

Reply:
@samgermain when we set `TICK_SIZE` as the precision mode, we add minimal precision fractions to precision, example 0.001 instead of 3